views/js/qtiCreator/widgets/static/figure/Widget.js

```diff
@@ -61,7 +63,7 @@ define([
     };
 
     FigureWidget.buildContainer = function buildContainer() {
-        if (this.element.attr('showFigure')) {
+        if (DISABLE_FIGURE_WIDGET || this.element.attr('showFigure')) {
```

Comment:
Since the default is to disabled (false), I think it should be a negation here, no?

```suggestion
        if (!DISABLE_FIGURE_WIDGET || this.element.attr('showFigure')) {
```